packages/neuron-ui/src/components/WalletWizard/hooks.ts

```diff
@@ -1,9 +1,9 @@
-import { useState, useCallback } from 'react'
+import { useState, useCallback, useEffect } from 'react'
 
 const MNEMONIC_SENTENCE_WORDS = 12
 
-export const useInputWords = () => {
-  const [inputsWords, setInputsWords] = useState<string[]>(new Array(MNEMONIC_SENTENCE_WORDS).fill(''))
+export const useInputWords = (wordsCount?: number) => {
```

Comment:
`wordCount = MNEMONIC_SENTENCE_WORDS` to avoid following `wordCount ?? MNEMONIC_SENTENCE_WORDS`